### PR TITLE
Add new k6 performance test forms subscribing

### PR DESCRIPTION
**Pull request by veljkho**
## Description

Added a new k6 test for subscribing to a front-end form.

## Code review notes

_N/A_

## QA notes

_N/A_

## Linked PRs

_N/A_

## Linked tickets

[MAILPOET-5874]

## After-merge notes

_N/A_

## Tasks

- [ ] I followed [best practices](https://codex.wordpress.org/I18n_for_WordPress_Developers) for translations
- [ ] I added sufficient test coverage
- [ ] I embraced TypeScript by either creating new files in TypeScript or converting existing JavaScript files when making changes

[MAILPOET-5874]: https://mailpoet.atlassian.net/browse/MAILPOET-5874?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: mailpoet/tests/performance/tests/forms-subscribing.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  waitAndType,
+  waitForSelectorToBeVisible,
+} from '../utils/helpers.js';
+
+export async function formsSubscribing() {
+  const page = browser.newPage();
+
+  try {
+    let subscriberEmail = 'blackhole+testform' + Date.now() + '@mailpoet.com';
+
```
Comment (triple0t) [inline mailpoet/tests/performance/tests/forms-subscribing.js:34]: I assume we are using the MailPoet sending service? 
The `blackhole` address works only with it.

**Review by triple0t — COMMENTED** [review]
Hi @veljkho 

Do we know how I can test this?

I'm getting this error `
ERRO[0002] (403/E4) Missing Authorization token or trying to run an anonymous test`

File: mailpoet/tests/performance/tests/forms-subscribing.js (hunk 1 above)
Comment (veljkho) [reply mailpoet/tests/performance/tests/forms-subscribing.js:34]: Yes, this test requires MSS so we will use blackhole ones.

**Comment by veljkho** [comment]
Hi @triple0t 
Thanks for reviewing my PR.

This test is for nightly using nightly test site and that would require you to add --url, --us and --pw parameters.

Currently, it is not possible to run specific test as before, writing the test path in the command, so [please comment out all other tests](https://d.pr/i/pDIsJk) from the nightly tests in your `scenarios.js` file, leaving the one you want to test only.

If you'd like to execute the test from your local, this would be the exact command:
`./do test:performance --scenario=nightlytests --url=https://mpperftesting.com --us=admin --pw=xxxxxx --head    `

Copy the xxxxxx - password from the secret store `secret_id=10626`

The error you're getting seems like if you have values in your .env file such as K6_CLOUD_TOKEN or K6_CLOUD_ID. Comment them out as well as you don't need those for local testing.

**Review by triple0t — APPROVED** [review]
LGTM. 
Thank you, @veljkho

**Pull request merged**